PROJECT

```diff
@@ -11,46 +12,118 @@ resources:
   domain: kubesphere.io
   group: logging
   kind: FluentBitConfig
-  path: kubesphere.io/fluentbit-operator/api/v1alpha2
+  path: kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2
   version: v1alpha2
 - api:
     crdVersion: v1
     namespaced: true
   domain: kubesphere.io
   group: logging
   kind: FluentBit
-  path: kubesphere.io/fluentbit-operator/api/v1alpha2
+  path: kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2
   version: v1alpha2
 - api:
     crdVersion: v1
     namespaced: true
   domain: kubesphere.io
   group: logging
   kind: Input
-  path: kubesphere.io/fluentbit-operator/api/v1alpha2
+  path: kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2
   version: v1alpha2
 - api:
     crdVersion: v1
     namespaced: true
   domain: kubesphere.io
   group: logging
   kind: Filter
-  path: kubesphere.io/fluentbit-operator/api/v1alpha2
+  path: kkubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2
   version: v1alpha2
 - api:
     crdVersion: v1
     namespaced: true
   domain: kubesphere.io
   group: logging
   kind: Output
-  path: kubesphere.io/fluentbit-operator/api/v1alpha2
+  path: kkubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2
   version: v1alpha2
 - api:
     crdVersion: v1
     namespaced: true
   domain: kubesphere.io
   group: logging
   kind: Parser
-  path: kubesphere.io/fluentbit-operator/api/v1alpha2
+  path: kkubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2
```

Comment:
modified to kubesphere.io

Reply:
Why is `kubesphere.io`? Shouldn't it be `fluent.io`?

Reply:
I see @wenchajun make this change in another pr :) /cc @wenchajun

Reply:
Is fluentd's domain also kubesphere.io, if so, it may need to be changed as well.

Reply:
will fix